go_pprof_scraper/datadog_checks/go_pprof_scraper/check.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, name, init_config, instances):
         self.url = self.instance.get("pprof_url")
         if not self.url:
             raise ConfigurationError("pprof_url is required")
+        if not self.url.endswith("/"):
+            # TODO: log a warning that we're mucking with the provided path?
```

Comment:
not sure a log line would be really useful, but it would not hurt if you're willing to add it

Reply:
Agreed, I think it wouldn't be useful after all. I'll remove the TODO.